core/src/main/resources/hudson/PluginManager/_table.css

```diff
@@ -8,6 +8,3 @@
     margin: 1em;
     text-align: right;
 }
-.compatWarning, .securityWarning {
```

Comment:
hm? Same as other classes?

Reply:
Oh, I thought they would be specific to the table and also just be usable there, since they are not in a global CSS file but in the PluginManager directory. Should this be changed as well?

Reply:
I think you just know CSS better than me :) If it is an internal CSS, my comment is definitely invalid. Thanks for clarification!